rsciio/utils/tools.py

```diff
@@ -503,27 +503,59 @@ def ensure_unicode(stuff, encoding="utf8", encoding2="latin-1"):
 
 
 def get_file_handle(data, warn=True):
-    """Return file handle of a dask array when possible; currently only hdf5 file are
-    supported.
     """
-    arrkey = None
+    Return file handle of a dask array when possible.
```

Comment:
How does this work with memmaped files? 

We should make a new issue to maybe make this a bit more routine.

Reply:
What if we put the original file into the metadata or added it to the signal dictionary.  Would that be slightly better as we could just track the initial object.

Reply:
Agree, we need an API to do that.